weather_mv/loader_pipeline/pipeline.py

```diff
@@ -14,33 +14,16 @@
 """Pipeline for reflecting lots of NetCDF objects into a BigQuery table."""
```

Comment:
Is this file level comment still correct?

Reply:
Thanks! Good catch.